src/App.jsx

```diff
@@ -1,31 +1,35 @@
-import React, { useState, useEffect } from 'react';
+import React, { useEffect, useState } from 'react';
 import { getAllData } from './util/index';
+import BookCardMain from '/src/components/BookCard/BookCardMain.jsx';
+import booksData from '/src/data/booksData.js';
 
 const URL = 'http://localhost:8000/api/v1/';
 
 function App() {
-  
-  const [message, setMessage] = useState(''); 
-
-  useEffect(() => {
-
-    (async () => {
-      const myData = await getAllData(URL)
-      setMessage(myData.data);
-    })();
-      
-    return () => {
-      console.log('unmounting');
-    }
-
-  }, []);
-
-  return (
-    <>
-      <h1>{message}</h1>
-    </>
-  );
-
+	const [message, setMessage] = useState('');
+
+	useEffect(() => {
+		(async () => {
+			const myData = await getAllData(URL);
+			setMessage(myData.data);
+		})();
+
+		return () => {
+			console.log('unmounting');
+		};
+	}, []);
+
+	const cards = booksData.map((item) => {
+		return <BookCardMain key={item.id} {...item} />;
+	});
+
+	return (
+		<div className="container">
+			{/* <Navbar /> */}
```

Comment:
Let's avoid leaving commented code lying around, it gets cluttered and can be difficult for others to know when it's safe to remove.